src/main/java/racing/view/OutputView.java

```diff
@@ -0,0 +1,22 @@
+package racing.view;
+
+import java.util.List;
+import racing.model.Cars;
+
+public class OutputView {
+
+    public static final String RESULT_LABEL = "실행 결과";
+
+    public static void printResultLabel() {
+        System.out.println(RESULT_LABEL);
+    }
+
+    public static void printResult(Cars cars) {
+        List<Integer> positions = cars.getPositions();
```

Comment:
```suggestion
    public static void printResult(Cars cars) {
        printResultLabel();
        List<Integer> positions = cars.getPositions();
        ...
     } 
     
     private static void printResultLabel() { ... }
```

노출되는 메소드는 하나로도 충분할 것 같아요!

Reply:
@ParkSeryu
생각해보니 이 부분은 제가 잘못 리뷰드린 것 같네요
해당 리뷰는 넘어가주세요~ 🙏